source/concurrent/Parallel.h

```diff
@@ -0,0 +1,149 @@
+/* Parallel.h
+Copyright (c) 2024 by tibetiroka
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef PARALLEL_H_
+#define PARALLEL_H_
+
+#include <version>
+
+
+
+// Only use this header if the underlying system doesn't support the standard algorithms.
+#ifndef __cpp_lib_parallel_algorithm
+
+// Mark that we are using a TaskQueue for parallel algorithms, instead of sdl functions.
+#define ES_PARALLEL_USE_TASK_QUEUE
+
+#include "TaskQueue.h"
```

Comment:
Was it supposed to be this way?
```suggestion
#include "TaskQueue.cpp"
```

Reply:
~The checks don’t seem to think so~ OK.

Reply:
The checks fail for a different reason. ~~I'm not exactly sure why.~~